ddtrace/contrib/starlette/patch.py

```diff
@@ -65,3 +50,30 @@ def unpatch():
     setattr(starlette, "_datadog_patch", False)
 
     _u(starlette.applications.Starlette, "__init__")
+
+    # Unwrap the handlers
+    _u(starlette.routing.Mount, "handle")
+    _u(starlette.routing.Route, "handle")
+
+
+def traced_handler(wrapped, instance, args, kwargs):
```

Comment:
Remove the helper function and use the get_argument_value to pull values out of args, kwargs, in this case scope receive and send. 

https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/internal/utils/__init__.py#L14